posixlib/src/main/resources/scala-native/errno.c

```diff
@@ -79,7 +79,11 @@ int scalanative_enfile() { return ENFILE; }
 
 int scalanative_enobufs() { return ENOBUFS; }
 
+#ifdef ENODATA
 int scalanative_enodata() { return ENODATA; }
+#else
+int scalanative_enodata() { return 0; }
+#endif
 
```

Comment:
Can we move pre-processor conditions to the actual body of the function? It's the convention used in most of the other sources and in my opinion it's more readable, even if it's a simple oneliner. 

```suggestion

int scalanative_enodata() { 
#ifdef ENODATA
    return ENODATA;
#else
    return 0; 
#endif
}

```

```